src/api/messages/getMessages.js

```diff
@@ -31,11 +31,15 @@ export type ServerReaction = $ReadOnly<{|
   |}>,
 |}>;
 
-export type ServerMessage = $ReadOnly<{|
```

Comment:
Let's keep the `$ReadOnly` here.

Reply:
Yep, `ServerMessageOf` (as you've done in the revision.) I.e. where the object-type syntax is. Then `ServerMessage` gets the benefit too, as it just uses `ServerMessageOf`.